frontend/src/pages/NutriScore.jsx

```diff
@@ -6,57 +6,49 @@ function NutriScore() {
-          En effet, le Nutri-Score met en évidence la valeur nutritionnelle d'un
-          produit de manière simple et visuelle, allant de la lettre A (vert)
-          pour les produits les plus sains à la lettre E (rouge) pour les moins
-          sains. Ce système aide les consommateurs à choisir des aliments plus
-          équilibrés et à opter pour des choix plus sains, en un coup d'œil sur
-          l'étiquette.
+          Nutritional values play an essential role in our understanding the
+          nutritional quality of food products. Nutri-Score, for example, is a
+          nutritional labeling system which uses these values to assign an
+          overall rating to a product food, thus facilitating consumer
+          decision-making. Indeed, the Nutri-Score highlights the nutritional
+          value of a produced in a simple and visual way, going from the letter
+          A (green) for the healthiest products with the letter E (red) for the
+          least healthy. This system helps consumers choose foods more balanced
+          and opt for healthier choices, at a glance the label.
         </p>
         <img className="text-line" src={Line} alt="line" />
       </div>
       <div className="text-container">
         <h2 className="text-title">
-          L'Évaluation des Valeurs Nutritionnelles dans le Nutri-Score
+          The Evaluation of Nutritional Values in the Nutri-Score
         </h2>
         <p className="text-description">
-          Le Nutri-Score prend en compte plusieurs valeurs nutritionnelles clés,
-          telles que les calories, les graisses saturées, les sucres, le sel,
-          les protéines, les fibres, ainsi que les pourcentages de la valeur
-          quotidienne (% VQ) de ces éléments. En combinant ces informations, le
-          Nutri-Score évalue la qualité nutritionnelle globale d'un aliment, en
-          prenant en compte les nutriments bénéfiques et les éléments à limiter.
-          Ce système d'étiquetage est devenu un outil précieux pour les
-          consommateurs soucieux de leur santé, les aidant à faire des choix
-          alimentaires plus éclairés dans un environnement où les informations
-          nutritionnelles peuvent parfois être complexes à interpréter.
+          The Nutri-Score takes into account several key nutritional values,
+          such as calories, saturated fats, sugars, salt, proteins, fibers, as
+          well as percentages of the value daily (% DV) of these elements. By
+          combining this information, the Nutri-Score evaluates the overall
+          nutritional quality of a food, in taking into account beneficial
+          nutrients and elements to limit. This labeling system has become a
+          valuable tool for health-conscious consumers, helping them make
+          choices more informed food in an environment where information
+          Nutritional information can sometimes be complex to interpret.
```

Comment:
Cadeau => The Nutri-Score takes into account several key nutritional values, such as calories, saturated fats, sugars, salt, proteins, fibers, as well as the percentages of the daily value (% DV) of these elements. By combining this information, the Nutri-Score assesses the overall nutritional quality of a food, considering both beneficial nutrients and elements to limit. This labeling system has become a valuable tool for health-conscious consumers, helping them make more informed dietary choices in an environment where nutritional information can sometimes be complex to interpret.